Xplat/src/main/resources/assets/botania/lang/zh_cn.json

```diff
@@ -2905,9 +2910,10 @@
 
   "botania.entry.sextant": "盖世六分仪",
   "botania.tagline.sextant": "帮助制作圆的工具",
-  "botania.page.sextant0": "在这个方块世界，艺术的创造往往不能避免圆，这时候他们不得不使用一种\"标记\"的邪恶技术弄一个圆出来。$(p)$(item)盖世六分仪$(0)是一个简单的——正如你所想——制造圆的工具。要使用它，只要简单点击作为圆心的方块和测量半径的方块即可。",
-  "botania.page.sextant1": "使用这个工具时会出现一个蓝圈，作为要制作的圆的简单预览。一旦设定好，一个虚拟的$(item)圆石$(0)圈会显现出来作为建筑参考。$(p)潜行右击能够重设工具并移除虚拟圆石",
+  "botania.page.sextant0": "在这个方块世界，创造一个圆形或球体几乎成了一门失传的手艺。以至于为了达成目的，他们不得不使用一种\"切换窗口\"的奇技淫巧弄一个圆出来。$(p)$(item)盖世六分仪$(0)作为一个制造圆的工具，为这种行为提供了一个简单的替代方案。",
+  "botania.page.sextant1": "手持这个工具时，潜行右键来选择模式。对准一个方块并长按右键来确定绘制的圆心或球心，然后望向远方来确定它的半径。此时会出现一个蓝圈，作为将要绘制的形状的简单预览。一旦设定好，一个虚拟的$(item)圆石$(0)圈会显现出来作为建筑参考。$(p)最后，潜行右键能够重设工具并移除虚拟圆石",
   "botania.page.sextant2": "你画了一个圆，使你充满了决心",
+  "botania.page.sextant_modes": "通常来讲，对于一个指定的正方形，有多种方法可以在内部绘制出圆形。六分仪可多次尝试从起点出发，移至正方形内任意一点从而绘制出不同大小的圆。$(p)如果二维形状还不能满足要求，当没有显示形状时，潜行右键可以切换形状模式。",
```

Comment:
Here I actually don't know what the original article is getting at. It seems to be describing the drawing of a circle inside a specified square, or the drawing of a ball inside a specified square? And it doesn't seem to be making an internal tangent circle? I'm just trying to use the description as best I can.

Reply:
I feel you, i was also struggling to translate that part.